PanModal/Controller/PanModalPresentationController.swift

```diff
@@ -133,7 +133,11 @@ open class PanModalPresentationController: UIPresentationController {
      */
     private lazy var dragIndicatorView: UIView = {
         let view = UIView()
-        view.backgroundColor = .lightGray
+        if let color = presentable?.dragIndicatorBackgroundColor {
+            view.backgroundColor = color
+        } else {
+            view.backgroundColor = .lightGray
+        }
```

Comment:
could we set this to `view.backgroundColor = presentable?.dragIndicatorBackgroundColor` - reason being that if we do not have a presentable object associated with the controller, we should assume that the view is not draggable, hence no backgroundColor